bundle/bundle.go

```diff
@@ -48,6 +52,17 @@ type Bundle struct {
 	Modules     []ModuleFile
 	Wasm        []byte // Deprecated. Use WasmModules instead
 	WasmModules []WasmModuleFile
+	Patch       Patch
+}
+
+// Patch contains an array of objects wherein each object represents the patch operation to be
+// applied to the bundle data.
+type Patch struct {
+	Data []types.PatchV1
+}
+
+func newPatch() Patch {
```

Comment:
This constructor seems unnecessary. The patch is kept as a value on the bundle and the data is a slice. The zero values for these types will be fine.

Reply:
Removed